### PR TITLE
LPD-41847 - Update filtersOrder property

### PR DESCRIPTION
**Pull request by juanjofgliferay**
https://liferay.atlassian.net/browse/LPD-41847

## Issue
The order of the filters in DSM was not persisted due to an error in the field name that stores the information.
- update property
- add tests in DSM
- add tests in the fragment

### CODE REVIEW

**Comment by juanjofgliferay** [comment]
ci:test:playwright

File: modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/pages/data_set/tabs/FiltersPage.ts [hunk 1]
```diff
@@ -173,6 +173,32 @@ export class FiltersPage {
 		await expect(visualFeedback).toContainText(text);
 	}
 
+	async assertTableCellContent({filterData, page, rowIndex = 0}) {
+
+		// await test.step('Assert table cell content', async () => {
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/pages/data_set/tabs/FiltersPage.ts:178]: :broom:

File: modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/pages/data_set/tabs/FiltersPage.ts [hunk 2]
```diff
@@ -173,6 +173,32 @@ export class FiltersPage {
...
+
+		const expectedRowContent = [
+			filterData.name,
+			filterData.fieldName,
+			filterData.type,
+		];
+
+		await expect(tableRowContent).toContainText(expectedRowContent);
+
+		// });
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/pages/data_set/tabs/FiltersPage.ts:198]: :broom:

File: modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/dateRangeFilters.spec.ts [hunk 3]
```diff
@@ -200,3 +200,105 @@ test('Can create Date-time filter without start and end dates', async ({
...
+		await expect(fdsFragmentPage.fdsFilterButton).toBeVisible();
+
+		await fdsFragmentPage.fdsFilterButton.click();
+		await fdsFragmentPage.page
+			.locator(`#${filterDropdownId}`)
+			.waitFor({state: 'visible'});
+		const filtersDropdown = fdsFragmentPage.page.locator(
+			`#${filterDropdownId}`
+		);
+		await expect(filtersDropdown.getByRole('menuitem')).toHaveCount(2);
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/dateRangeFilters.spec.ts:266]: `filtersDropdown.getByRole('menuitem')` repeats a lot of times, so we can extract it to a local variable or page property.

File: modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/dateRangeFilters.spec.ts [hunk 4]
```diff
@@ -200,3 +200,105 @@ test('Can create Date-time filter without start and end dates', async ({
...
+			dataSetLabel,
+			layout,
+		});
+	});
+
+	const filterDropdownId =
+		await fdsFragmentPage.fdsFilterButton.getAttribute('aria-controls');
+
+	await test.step('Assert that the date filters are in the UI', async () => {
+		await expect(fdsFragmentPage.fdsFilterButton).toBeVisible();
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/dateRangeFilters.spec.ts:257]: Not necessary for this PR, but we should at some point rename `FDSFragmentPage` to `DataSetFragmentPage`, and remove `fds*` prefix from all properties inside.

**Review by markocikos — COMMENTED** [review]
@juanjofgliferay looks good, some minor nits inline

File: modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/dateRangeFilters.spec.ts (hunk 4 above)
Comment (juanjofgliferay) [reply modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/dateRangeFilters.spec.ts]: No problem. It is done in a separate commit

**Review by markocikos — APPROVED** [review]
@juanjofgliferay LGTM! :sparkles:

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:playwright

**Comment by juanjofgliferay** [comment]
ci:forward

File: modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/Filters.tsx [hunk 5]
```diff
@@ -269,12 +269,12 @@ function Filters({
 
 		const responseJSON = await response.json();
 
-		const storedFDSFiltersOrder = responseJSON?.fdsFiltersOrder;
+		const storedFDSFiltersOrder = responseJSON?.filtersOrder;
```
Comment (dsanz) [inline modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/Filters.tsx:272]: we can also take this opportunity to remove `FDS` from here as well: `storedFiltersOrder`

**Review by dsanz — APPROVED** [review]
LGTM ;)

**Comment by juanjofgliferay** [comment]
ci:test:relevant

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:relevant

**Comment by juanjofgliferay** [comment]
ci:test:relevant

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:relevant

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:relevant

**Pull request closed without merging**